system/LINK/etc/default/evm

```diff
@@ -28,3 +28,12 @@ export KEY_ROOT=/var/www/miq/vmdb/certs
 [[ -s /opt/rh/postgresql92/enable ]] && source /opt/rh/postgresql92/enable
 [[ -s /opt/rh/nodejs010/enable ]] && source /opt/rh/nodejs010/enable
 [[ -s /etc/default/evm_productization ]] && source /etc/default/evm_productization
+
+if [[ -s /opt/rh/ruby200/enable ]]
+  source /opt/rh/ruby200/enable
+else
+  rubies="/opt/rubies/ruby-2.0.0*/bin"
+  # add latest (alphabetically) ruby version
+  export PATH=$PATH:${rubies[${#rubies[@]} - 1]}
+fi
```

Comment:
No, @kbrock.

We want a new file: system/LINK/etc/default/evm_productization in upstream
It should contain the ruby-install path:

```
rubies="/opt/rubies/ruby-2.0.0*/bin"
# add latest (alphabetically) ruby version
export PATH=$PATH:${rubies[${#rubies[@]} - 1]}
````

We'll handle the rest elsewhere using productization overlays.